app/helpers/application_helper/toolbar/ems_infra_center.rb

```diff
@@ -143,7 +143,7 @@ class ApplicationHelper::Toolbar::EmsInfraCenter < ApplicationHelper::Toolbar::B
         button(
           :ems_native_console,
           'pficon pficon-screen fa-lg',
-          N_('Open a native console for this cloud provider'),
+          N_('Open a native console for this Infrastructure Provider'),
```

Comment:
For consistency, I think the case should match

```suggestion
          N_('Open a native console for this infrastructure provider'),
```

Reply:
I just copied what was done for other messages in the same file:
> Show Timelines for this Infrastructure Provider
> Open Admin UI for this Infrastructure Provider

etc...
But I agree it should be lowercase.